sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -2681,6 +2747,11 @@ public partial class RangeFacetResult<T> where T : struct
         public T? From { get { throw null; } }
         public T? To { get { throw null; } }
     }
+    public partial class ResettingSkills
```

Comment:
This doesn't read like a .NET class name

Reply:
This is a class with just one `IList` property in it. With the current name on the service-side (`skillNames`), it is hard to imagine more properties being added.
Anyways, I was thinking of other options - `SkillsToReset`, `SkillsBeingReset`.